index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./app.css">
+    <title>Box Model</title>
+  </head>
+  <body>
+
+    <div class="navbar">
+      <h3 class="navbar__left">Popular Github</h3>
+      <a href="#" class="navbar__right">People</a>
+      <a href="#" class="navbar__right">Repos</a>
+    </div>
+
+<div class="content">
+<div class="column-1">
```

Comment:
Don't use numbered elements to divide the grid items. Each user profile should be able to use the same classes for each element and just the img src and text will change.